src/components/Footer/index.js

```diff
@@ -2,9 +2,15 @@ import React from 'react'
 
 import './style.css'
 
+import { SocialIcon } from 'react-social-icons';
+
 const Footer = () => {
   return (
-    <footer className="App-footer" id="footer">Footer</footer>
+    <footer className="App-footer" id="footer">
+      <div className= "row">
+      <div className = "col-lg-12 col-md-12 col-sm-12">&copy; Copyright <strong>Created by Jon, Ansar, Jade & Iain</strong> Contact Us: <SocialIcon url="https://github.com/JonHarrison/the-book-nook.git" /></div>
```

Comment:
Maybe put the copyright and contact us bits in their own divs so that they can be styled with a margin to separate from each other ? Not necessarily for now.

Reply:
Exactly what I was thinking - I'll add it as an issue for later especially now I know what I'm doing a little bit more. Thank you